sdk/servicebus/service-bus/README.md

```diff
@@ -3,10 +3,11 @@
 Azure Service Bus is a highly-reliable cloud messaging service from Microsoft
 
 Use the client library for Azure Service Bus in your Node.js application to
+
 - Send messages to a Queue or Topic
 - Receive messages from a Queue or Subscription
 
-[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus) | [Package (npm)](https://www.npmjs.com/package/@azure/service-bus) | [Product documentation](https://azure.microsoft.com/en-us/services/service-bus/)
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus) | [Package (npm)](https://www.npmjs.com/package/@azure/service-bus) | [API Reference Documentation](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/?view=azure-node-preview) | [Product documentation](https://azure.microsoft.com/en-us/services/service-bus/)
```

Comment:
Will we need to change this once it is live?

Reply:
That "preview" is the fact that the sdk is in preview, not the docs being in preview.
These are live docs.
The preview docs would start with `review.docs.microsoft.com` or something similar